sensor/kubernetes/complianceoperator/updater_test.go

```diff
@@ -75,6 +72,9 @@ func (s *UpdaterTestSuite) TearDownTest() {
 }
 
 func (s *UpdaterTestSuite) TestDefaultNamespace() {
+	// Prepend a SelfSubjectAccessReview reactor to report write access.
+	s.prependSSAReactorToFakeClient(true)
```

Comment:
I think it looks better to specifically call `prependSSAReactorToFakeClient` at the beginning of each test instead of in the setup. Notice that in some tests, `prependSSAReactorToFakeClient` is called again with `false`. This way it is only called once per test.